fetch-utils.js

```diff
@@ -37,6 +37,31 @@ export async function logout() {
     return (window.location.href = '../');
 }
 
-// function checkError({ data, error }) {
-//     return error ? console.error(error) : data;
-// }
+// everything given is above, functions I am creating are below
+
+
+
+export async function fetchItems() {
+    const resp = await client.from('todo_shopping').select().order('id');
+// console.log('hi', resp);
+    return checkError (resp);
+}
```

Comment:
Nice job using order by id, as ordering by complete would change the order based on the user clicking the shopping item.